x/bank/v2/module.go

```diff
@@ -94,18 +94,13 @@ func (am AppModule) ExportGenesis(ctx context.Context) (json.RawMessage, error)
 // RegisterMsgHandlers registers the message handlers for the bank module.
 func (am AppModule) RegisterMsgHandlers(router appmodulev2.MsgRouter) {
 	handlers := keeper.NewHandlers(am.keeper)
-
-	appmodulev2.RegisterMsgHandler(router, handlers.MsgUpdateParams)
-	appmodulev2.RegisterMsgHandler(router, handlers.MsgSend)
-	appmodulev2.RegisterMsgHandler(router, handlers.MsgMint)
```

Comment:
was this the wrong spot for this registration logic? registration of services is usually handled in module.go, so curious if there was an issue with this

Reply:
this is just so you could do keeper testing where you register to a router at the `keeper` package level and don't have to import `module`, creating some kind of circular dep